cryptus-app/components/wallet_viewer/profile_wallet_viewer/collection_block/collection_nfts/collection_nfts.tsx

```diff
@@ -5,48 +5,62 @@ import s from "./collection_nfts.module.scss";
 
 //internal imports
 import NFT_picture_block_in_wallet from "./nft_picture_block_in_wallet/nft_picture_block_in_wallet"
-import { nft } from "../../../../../lib/data_types"
+import { nft, nft_collection } from "../../../../../lib/data_types"
 
+  export default function CollectionNFTs(props: { nft: nft[] }) {
+    const empty_nft = {
+      id: 0,
+      collection: "",
+      name: "",
+      description: "",
+      external_url: "",
+      image_url: "data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=",
+      properties: '',
+      last_sale_price: 0,
+      last_sale_symbol: "",
+      rarity_rank: 0,
+    } as nft
 
-export default function Collection_NFTs(props: { nft: nft }) {
-  const { nft } = props;
-  
+    const { nft } = props;
+    const nfts = [
+      empty_nft,
+      empty_nft,
+      empty_nft,
+      empty_nft,
+      empty_nft,
+      empty_nft,
+      empty_nft,
+    ]
+
+    for (let index = 0; index < nft.length; index++) {
+      nfts[index] = nft[index];      
+    }
```

Comment:
no for loop, use .foreach instead.
But do you really need to do that, can't you use an array of nfts passed through props?

Reply:
nfts[index] = nft[index] lol

Reply:
J'avais oublié de le changer haha